src/main/scala/fr/polytechnique/cmap/cnam/etl/extractors/mco/McoExtractor.scala

```diff
@@ -32,19 +32,43 @@ trait McoExtractor[EventType <: AnyEvent] extends Extractor[EventType] with McoS
     Seq(eventBuilder[EventType](patientId, groupId, code(row), weight, eventDate, endDate))
   }
 
+  /** It gets columnName's value from row.
+   *
+   * @return A string value.
+   */
   def code = (row: Row) => row.getAs[String](columnName)
 
+  /** It gets PatientID value from row.
+   *
+   * @param r The row itself.
+   * @return The value of PatientID.
+   */
   def extractPatientId(r: Row): String = {
     r.getAs[String](ColNames.PatientID)
   }
 
+  /** It gets EtaNum value, RsaNum value and Year from row.
+   *
+   * @param r The row itself.
+   * @return The value of groupId.
+   */
   override def extractGroupId(r: Row): String = {
     r.getAs[String](ColNames.EtaNum) + "_" +
       r.getAs[String](ColNames.RsaNum) + "_" +
       r.getAs[Int](ColNames.Year).toString
   }
 
+  /** It gets EstimatedStayStart from row.
```

Comment:
_Extracts the EstimatedStayStart as the start._

You will need to document the EstimatedStayStart method, as this an artefact of our code and does not exist as is in the MCO Source.